base_report_to_printer/wizards/print_attachment_report.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+import base64
+
+from openerp import models, api, fields
+
+
+class PrintAttachment(models.TransientModel):
+    _name = 'wizard.print.attachment'
+    _description = 'Print Attachment'
+
+    printer_id = fields.Many2one(
+        comodel_name='printing.printer', string='Printer', required=True,
+        help='Printer used to print the attachments.'
+    )
+    attachment_line_ids = fields.One2many(
+        'wizard.print.attachment.line', 'wizard_id',
+        string='Attachments to print',
+    )
+
+    @api.multi
+    def print_attachments(self):
+        """ Prints a label per selected record """
+        self.ensure_one()
+        for att_line in self.attachment_line_ids:
+            content = base64.b64decode(att_line.attachment_id.datas)
```

Comment:
Should we test that `attachment_id.datas` could return `False` (not required on `ir.attachment`)?

Reply:
Not sure it will happen, it's not required because it's computed using `store_fname` or `db_datas` but I added a warning message anyway. :wink: